src/renderers/dom/fiber/ReactDOMFiberEntry.js

```diff
@@ -163,6 +163,22 @@ function shouldAutoFocusHostComponent(type: string, props: Props): boolean {
   return false;
 }
 
+// TODO: Better polyfill
```

Comment:
It's fine. Maybe we should colocate it with `ReactDOMFrameScheduling` to have all these in one place?

It is also `performance.now` aware: https://github.com/facebook/react/blob/master/src/renderers/shared/ReactDOMFrameScheduling.js#L73